src/sst/core/configGraph.cc

```diff
@@ -30,6 +31,48 @@ namespace SST {
 
 // static bool zero_latency_warning = false;
 
+// Functions to check component and link names
+static int   bad_comp_name_count = 0;
+static int   bad_link_name_count = 0;
+static regex valid_name_regex =
+    regex("((([_][a-zA-Z0-9])|[a-zA-Z])[_a-zA-Z0-9]*)(.(([_][a-zA-Z0-9])|[a-zA-Z])[_a-zA-Z0-9]*)*");
+
+static const int max_invalid_name_prints = 10;
```

Comment:
This can probably be `constexpr` unless you want to keep it similar to the `bad_...`.